app/code/Magento/Bundle/Test/Unit/Pricing/Price/ConfiguredPriceTest.php

```diff
@@ -0,0 +1,182 @@
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->product->expects($this->once())->method('getPriceInfo')->willReturn($this->priceInfo);
+        $this->product->expects($this->any())->method('getId')->willReturn(123);
+
+        $this->item = $this->getMockBuilder(ItemInterface::class)
+            ->getMock();
+        $this->item->expects($this->any())->method('getProduct')->willReturn($this->product);
+
+        $this->priceCurrencyMock = $this->getMockForAbstractClass(PriceCurrencyInterface::class);
+
+        $this->jsonSerializerMock = $this->getMockBuilder(Json::class)
+            ->getMock();
+        $this->configuredPriceSelectionMock = $this->getMockBuilder(ConfiguredPriceSelection::class)
+            ->setMethods(['getSelectionPriceList'])
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->configuredPriceSelectionMock->expects($this->any())->method('getSelectionPriceList')
+            ->willReturn($this->prepareAndReturnSelectionPriceDataStub());
+        $this->amountInterfaceMock = $this->getMockBuilder(AmountInterface::class)->getMock();
+        $this->amountInterfaceMock->expects($this->any())->method('getBaseAmount')
+            ->willReturn(100.0);
+        $this->calculator = $this->getMockBuilder(Calculator::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->calculator->expects($this->any())->method('calculateBundleAmount')
+            ->willReturn($this->amountInterfaceMock);
+        $this->discountCalculator = $this->getMockBuilder(DiscountCalculator::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->discountCalculator->expects($this->any())->method('calculateDiscount')
+            ->willReturn(-5.0);
+        $this->model = new ConfiguredPrice(
+            $this->product,
+            1,
+            $this->calculator,
+            $this->priceCurrencyMock,
+            null,
+            $this->jsonSerializerMock,
+            $this->configuredPriceSelectionMock,
+            $this->discountCalculator,
+        );
+        $this->model->setItem($this->item);
+    }
+
+    private function prepareAndReturnSelectionPriceDataStub()
+    {
+        $first = new DataObject();
+        $first->setValue(2);
+        $first->setQuantity(1);
+        $second = new DataObject();
+        $second->setValue(3);
+        $second->setQuantity(1);
+        return [
+            $first,
+            $second
+        ];
+    }
+
+    /**
+     * Test of value getter when item presented
+     */
+    public function testGetValueMethod()
+    {
+        $valueFromMock = $this->model->getValue();
+        $this->assertEquals(95., $valueFromMock);
+    }
+
+    /**
+     * Test of value getter if no product item
+     */
+    public function testGetValueMethodNoItem()
+    {
+        unset($this->item);
```

Comment:
Could you please clarify the reason of removing and re-creating the `$this->item` object? Usually for a better structure and clearance, the needed objects should be instantiated within the test case.